cluster-scope/base/operators.coreos.com/subscriptions/grafana-operator/subscription.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: grafana-operator
+  namespace: openshift-grafana-operator
```

Comment:
you can deploy this in your own namespace so that you don't disturb the rest of the cluster with it.
[This](https://github.com/operate-first/apps/blob/master/cluster-scope/base/operators.coreos.com/subscriptions/opf-grafana/kustomization.yaml#L3) is how you could specify the namespace in your kustomization.